app/views/companies/index.html.haml

```diff
@@ -14,26 +14,40 @@
       #{t('.h_companies_listed_below')}
       #{t('.how_to_search')}
 
-  %table
-    %thead
-      %tr
-        %th= t('.category')
-        %th= t('.name')
-        %th= t('.region_land')
-        - if current_user.try(:admin)
-          %th= t('.company_number')
+  .panel.panel-default{ style: 'padding-left: 5px; padding-right: 5px;' }
+    = render 'search_form'
+
+  - if @companies.empty?
+    %strong No records matched your search criteria.
```

Comment:
THIS needs to be t()

"Din sökning gav inget resultat, prova att ta bort något av dina val."